R/util.R

```diff
@@ -146,3 +146,33 @@ cyphr_file <- function(...) {
 file_copy <- function(...) {
   stopifnot(file.copy(...))
 }
+
+guess_key_options <- function(error = FALSE) {
```

Comment:
I don't love this bit - particularly that the error is a different sort of thing. Perhaps:

```
guess_key_error <- function() {
  paste(sprintf("%s[.pub] pair", guess_key()), collapse = " or ")
}
```

replacing use of `guess_key_options(error = TRUE)`

Later we might want to generalise this approach and allow setting key priorities?

Reply:
Agree - have separated that out so we have one function providing the list of key-stubs and separate error func...